### ISSUE TITLE
Fix error when committing in Windows OS

### ISSUE THREAD

**Issue by silentninja** [silentninja/easygit#3]
@Kristinita Please use this issue to update on the problem you are facing

**Comment by silentninja** [comment]
Please try out the latest commit which should fix #2 and let me know what the the error dialog shows. Also any related message in the console would be helpful

**Comment by Kristinita** [comment]
I reinstall easygit →  I made changes to a file in my repository → 

```
key evt: shift+super+g
command: add_then_commit_then_push {"targeted": "all"}
error: This folder is not a git repository! Make it one and try again
```

![Not a git repository](http://i.imgur.com/fczSY3Y.png)

But I am sure that this file is part of my repository on GitHub. I made a change to a different file from other my repository, but I got the same error.

Thanks.

**Comment by silentninja** [comment]
I dont have a windows machine to test it out. It works fine in mac and linux though. I had reverted back the repo check. Please try out the fix in #4

**Comment by Kristinita** [comment]
Don't push:

```
command: add_then_commit_then_push {"targeted": "repo"}
Traceback (most recent call last):
  File "commands in E:\Sublime Text 3\Data\Installed Packages\easygit.sublime-package", line 55, in on_done
  File "functions in E:\Sublime Text 3\Data\Installed Packages\easygit.sublime-package", line 34, in git
TypeError: check_output() got multiple values for keyword argument 'stderr'

```

```
command: add_then_commit_then_push {"targeted": "file"}
Traceback (most recent call last):
  File "commands in E:\Sublime Text 3\Data\Installed Packages\easygit.sublime-package", line 55, in on_done
  File "functions in E:\Sublime Text 3\Data\Installed Packages\easygit.sublime-package", line 34, in git
TypeError: check_output() got multiple values for keyword argument 'stderr'
```

**Comment by silentninja** [comment]
My bad, I  had specified a keword argument as well as passing one in kwargs. It correctly took the keyword argument instead of the one in kwargs in my computer which made me miss the problem.Fixed it in the latest commit

**Comment by Kristinita** [comment]
@silentninja , nice, it's work for me!

Do you plan to add your plugin in Package Control? Perhaps, under a different name to distinguish it from Ivanca plugin.

Thanks.

**Comment by silentninja** [comment]
The original repo owner has merged the PR i had submitted Ivanca/easygit@177ca4f4c95d37003373bd4c6f0ea12c96ff1c64, please continue using the original one
